src/helper_gitlab_modify.sh

```diff
@@ -260,45 +260,44 @@ git_pull_gitlab_repo() {
 create_empty_repository_v0() {
   local gitlab_repo_name="$1"
   local gitlab_username="$2"
-   
+
    # load personal_access_token (from hardcoded data)
     personal_access_token=$(echo "$GITLAB_PERSONAL_ACCESS_TOKEN" | tr -d '\r')
-  
+
   # TODO(a-t-0): Check if GitLab server is running.
-  
+
   # Check if repository already exists in GitLab server.
   if [ "$(gitlab_mirror_repo_exists_in_gitlab "$gitlab_repo_name")" == "FOUND" ]; then
-  
+
     # If it already exists, delete the repository
     delete_existing_repository "$gitlab_repo_name" "$gitlab_username"
     sleep 30
-    
+
     # Verify the repository is deleted.
     if [ "$(gitlab_mirror_repo_exists_in_gitlab "$gitlab_repo_name")" == "FOUND" ]; then
       # Throw an error if it is not deleted.
       echo "The GitLab repository was supposed to be deleted, yet it still exists."
       #exit 177
     fi
   fi
-  
+
   # Create repository.
   curl -H "Content-Type:application/json" "$GITLAB_SERVER_HTTP_URL/api/v4/projects?private_token=$personal_access_token" -d "{ \"name\": \"$gitlab_repo_name\" }"
   sleep 30
-  
+
   # Verify the repository is created.
   if [ "$(gitlab_mirror_repo_exists_in_gitlab "$gitlab_repo_name")" != "FOUND" ]; then
     # Throw an error if it is not created succesfully.
     echo "The GitLab repository was supposed to be created, yet it does not yet exists."
     exit 178
   fi
-  
+
 }
 
 
 #######################################
 # Checks if repository exists in the GitLab server and creates a new empty 
-# repository 
-# if that is not the case.
+# repository.
```

Comment:
I think this should be extend to something like:
```
# Checks if repository exists in the GitLab server and creates a new empty
# repository if it does not yet exist.
```
I think this is important because the current/latest description is not an accurate description of what the function does. Because it implies the function always creates a new empty repository, whereas in reality, the function only creates the repository if it does not exist already.